kong/db/declarative/init.lua

```diff
@@ -802,6 +802,38 @@ function declarative.load_into_cache_with_events(entities, meta, hash)
     return nil, "failed to persist cache page number inside shdict: " .. err
   end
 
+  local worker_count = ngx.worker.count()
+  local sleep_left = 60
+  local sleep_time = 0.0375
+
+  while true do
+    local flips = ngx.shared.kong:get("declarative:flips")
```

Comment:
Since this key is set inside the `flip_config` worker event and it is executed asynchronously. Is there a possibility that this may return `nil` because the worker event haven't been executed by any other workers yet?

Reply:
The post above will execute it right away on current worker. Other workers process it via timer. post is actually post+poll.